java/ql/src/experimental/Security/CWE/CWE-927/SensitiveBroadcast.ql

```diff
@@ -0,0 +1,172 @@
+/**
+ * @name Broadcasting sensitive data to all Android applications
+ * @id java/sensitive-broadcast
+ * @description An Android application uses implicit intents to broadcast sensitive data to all applications without specifying any receiver permission.
+ * @kind path-problem
+ * @tags security
+ *       external/cwe-927
+ */
+
+import java
+import semmle.code.java.frameworks.android.Intent
+import semmle.code.java.dataflow.TaintTracking
+
+/**
+ * Gets a regular expression for matching common names of variables that indicate the value being held contains sensitive information.
+ */
+private string getCommonSensitiveInfoRegex() {
```

Comment:
Move this to a new file in https://github.com/github/codeql/tree/main/java/ql/src/semmle/code/java/security (perhaps `SensitiveInfo.qll`) and use that from here and the sensitive logging query that uses the same regex.

Reply:
Moved the common one <code>getCommonSensitiveInfoRegex()</code> to <code>java/ql/src/semmle/code/java/security/SensitiveActions.qll</code>